RELEASE-NOTES.txt

```diff
@@ -5,6 +5,7 @@
 * [**] Media: Fix Atomic sites video not playing issue. (https://github.com/wordpress-mobile/WordPress-Android/pull/15285)
 * [*] Block editor: Fixes bug preventing button block gradient background from being updated in the editor [https://github.com/wordpress-mobile/WordPress-Android/pull/15312]
 * [*] Updated the wording for the "Posts" and "Pages" entries in My Site screen [https://github.com/wordpress-mobile/WordPress-Android/pull/15327]
+* [*] After navigating through a couple of related posts in Reader, load the correct post after rotation [https://github.com/wordpress-mobile/WordPress-Android/issues/14195]
```

Comment:
@dnalves Thanks for making an entry here! For these entries we try to write them in the past tense when possible. So something like this would suffice: "Fixed an issue where navigating through a couple of related posts in Reader caused the incorrect post to be loaded after rotation". Let me know what you think. 

Our release notes are sent to the Editorial team, so they are reviewed before being published to the Play Store.

Reply:
Makes a lot of sense! Thanks for the suggestion. And I'm glad those are reviewed, I wouldn't trust myself with final copies 😂 

(I'll try to use Github's code suggestion feature to fix it here)

```suggestion
* [*] Fixed an issue where navigating through a couple of related posts in Reader caused the incorrect post to be loaded after rotation [https://github.com/wordpress-mobile/WordPress-Android/issues/14195]
```

Reply:
Thanks @dnalves 😄